ios/Capacitor/Capacitor/CAPFile.swift

```diff
@@ -11,7 +11,7 @@ public class CAPFile {
  * disk, in a bundle, or in another location.
  */
 @objc public class CAPFileManager: NSObject {
-    static func get(path: String) -> CAPFile? {
+    public static func get(path: String) -> CAPFile? {
```

Comment:
I think this would benefit from a more descriptive name. This isn't getting "getting" anything; CAPFileManager doesn't store data. This is transforming the parameter that gets passed in but it's impossible to tell what it is expected do to from the name.